dnstable_convert.c

```diff
@@ -323,9 +331,11 @@ process_rdata_name_rev(Nmsg__Sie__DnsDedupe *dns, size_t i, ubuf *key, ubuf *val
 		break;
 	case WDNS_TYPE_MX:
 		offset = 2;
+		len -= offset;
 		break;
 	case WDNS_TYPE_SRV:
 		offset = 6;
+		len -= offset;
 		break;
 	default:
```

Comment:
is the default: case an error or are the other rrtypes that exist that we simply don't need to handle?

Reply:
The `default:` case here is that we do not create an index entry for the rdata.

Reply:
please add a comment for that.